simulation-construction-set/simple-pendulum-with-controller/build.gradle.kts

```diff
@@ -0,0 +1,16 @@
+plugins {
+   id("us.ihmc.ihmc-build") version "0.20.2"
+}
+
+ihmc {
+   group = "us.ihmc"
+   version = "0.0"
+   vcsUrl = "https://github.com/ihmcrobotics/ihmc-open-robotics-software-tutorials"
+   openSource = true
+
+   configureDependencyResolution()
+   configurePublications()
+}
+
+mainDependencies {
+}
```

Comment:
Same as for the falling brick, you need `api("us.ihmc:simulation-construction-set:0.16.0")`